modules/installation/pages/install-server-unified.adoc

```diff
@@ -27,7 +25,7 @@ Before installing {susemgr}, ensure your physical or virtual machine has enough
 . When prompted, select [guimenu]``Installation``.
 +
 image::unifiedinstaller-install.png[scaledwidth=80%]
-. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check the [guimenu]``SUSE Manager Server`` checkbox, and click btn:[Next].
+. In the [guimenu]``Language, Keyboard and Product Selection`` screen, select the [guimenu]``SUSE Manager Server`` radio button, and click btn:[Next].
```

Comment:
That doesn't meet our style (https://github.com/uyuni-project/uyuni-docs/wiki/wug#verbs)

This works better:

```suggestion
. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check [guimenu]``SUSE Manager Server``, and click btn:[Next].
```